servicecatalog_factory/cli_test.py

```diff
@@ -222,3 +222,144 @@ def test_product_exists_when_it_doesnt(mocker, sut):
 
     # verify
     assert actual_result == expected_result
+    
+
+def test_merge_simple_case(sut):
```

Comment:
can you please combine the merge tests together.

You can use https://docs.pytest.org/en/latest/parametrize.html to reduce the lines of code